scripts/check-tsc.ts

```diff
@@ -0,0 +1,22 @@
+import { execSync, type SpawnSyncReturns } from "node:child_process";
+import { findDirWithFileRecursive } from "./find.js";
+
+async function main(): Promise<void> {
+  for (const project of await findDirWithFileRecursive("tsconfig.json"))
+    try {
+      console.log(`🔄 ${project}`);
+      execSync("npx tsc --noEmit", {
```

Comment:
The checks are currently async to speed things up, so you'll want to `await Promise.all(projects.map(async (project) => { ... }))`

Reply:
Done.